pkg/controller/job/metrics/metrics.go

```diff
@@ -64,9 +64,9 @@ var (
 	JobFinishedNum = metrics.NewCounterVec(
 		&metrics.CounterOpts{
 			Subsystem:      JobControllerSubsystem,
-			Name:           "job_finished_total",
-			Help:           "The number of finished job",
-			StabilityLevel: metrics.ALPHA,
+			Name:           "jobs_finished_total",
```

Comment:
To be consistent with the terminology used in the status API (https://github.com/kubernetes/api/blob/v0.25.3/batch/v1/types.go#L326-L401), would it make sense to rename this to `jobs_completed_total`?

Reply:
No, because it includes both Complete and Failed jobs.

Reply:
oh ok, thank you for clarifying :)